### ISSUE TITLE
Issue when exporting the frozen graph model

### ISSUE THREAD

**Issue by mazzz56** [mozilla/DeepSpeech#1341]
Command: 
python DeepSpeech.py --n_hidden 2048 --checkpoint_dir fine_tuning_checkpoints/ --epoch 100 --train_files /home/moazam/DeepSpeech-master/bin/output/cv-valid-train.csv --dev_files /home/moazam/DeepSpeech-master/bin/output/cv-valid-dev.csv --test_files /home/moazam/DeepSpeech-master/bin/output/cv-valid-test.csv --learning_rate 0.0001 --export_dir /home/moazam/DeepSpeech-master/output

Error:
W Parameter --validation_step needs to be >0 for early stopping to work
I STARTING Optimization
I Training of Epoch 99 - loss: 0.000032
I FINISHED Optimization - training time: 0:02:30
Loading the LM will be faster if you build a binary file.
Reading data/lm/lm.binary
----5---10---15---20---25---30---35---40---45---50---55---60---65---70---75---80---85---90---95--100
terminate called after throwing an instance of 'lm::FormatLoadException'
  what():  native_client/kenlm/lm/read_arpa.cc:65 in void lm::ReadARPACounts(util::FilePiece&, std::vector<long unsigned int>&) threw FormatLoadException.
first non-empty line was "version https://git-lfs.github.com/spec/v1" not \data\. Byte: 43
Aborted (core dumped)

need to ask how to get rid off this error.

**Comment by lissyx** [comment]
Please read the documentation and setup git-lfs properly.

**Comment by lissyx** [comment]
The error obviously shows that your `data/lm` directory is not properly populated, so you might have to force some `git lfs pull` or something like that. Worst case, re-clone.

**Comment by mazzz56** [comment]
what do mean by directory is not properly populated

**Comment by mazzz56** [comment]
i have checked all lm.binary and trie the are 1.6GB and 43.7MB respectively but still we are getting the error

**Comment by mazzz56** [comment]
@lissyx 
![1111](https://user-images.githubusercontent.com/36292053/39519284-9521b94c-4e1f-11e8-9bcd-ea75e90f7f12.png)

**Comment by mazzz56** [comment]
@lissyx 
![1111](https://user-images.githubusercontent.com/36292053/39519447-367de11c-4e20-11e8-96b7-372c81b5ef65.png)

i am running this following command but cannot remove these error as i cannot understand these errors atleast explain me these errors
